precompile/asset-metadata-extender/src/mock.rs

```diff
@@ -26,34 +20,32 @@ frame_support::construct_runtime!(
 	}
 );
 
+type AccountId = H160;
+type Balance = u64;
+type Block = frame_system::mocking::MockBlock<Test>;
+
+#[derive_impl(frame_system::config_preludes::TestDefaultConfig as frame_system::DefaultConfig)]
 impl frame_system::Config for Test {
-	type BaseCallFilter = frame_support::traits::Everything;
-	type BlockWeights = ();
-	type BlockLength = ();
-	type DbWeight = RocksDbWeight;
-	type RuntimeOrigin = RuntimeOrigin;
-	type RuntimeCall = RuntimeCall;
-	type Nonce = u64;
-	type Hash = H256;
-	type Hashing = BlakeTwo256;
+	type Block = Block;
 	type AccountId = AccountId;
 	type Lookup = IdentityLookup<Self::AccountId>;
-	type Block = Block;
-	type RuntimeEvent = RuntimeEvent;
-	type BlockHashCount = ConstU64<250>;
-	type Version = ();
-	type PalletInfo = PalletInfo;
-	type AccountData = AccountData<u64>;
-	type OnNewAccount = ();
-	type OnKilledAccount = ();
-	type SystemWeightInfo = ();
-	type SS58Prefix = ConstU16<42>;
-	type OnSetCode = ();
-	type MaxConsumers = frame_support::traits::ConstU32<16>;
+	type DbWeight = RocksDbWeight;
+	type AccountData = pallet_balances::AccountData<Balance>;
 }
 
-type AccountId = H160;
-type Block = frame_system::mocking::MockBlock<Test>;
+parameter_types! {
+	pub const ExistentialDeposit: u64 = 0;
+}
+
+// Pallet Balances
+#[derive_impl(pallet_balances::config_preludes::TestDefaultConfig as pallet_balances::DefaultConfig)]
+impl pallet_balances::Config for Test {
+	type ExistentialDeposit = ExistentialDeposit;
+	type AccountStore = System;
+	type RuntimeHoldReason = ();
+	type DustRemoval = ();
```

Comment:
why did you keep some types like `DustRemoval` but remove other ones? I ask because default implementation of `DustRemoval` in `TestDefaultConfig` is also `()`?